lib/base_service.ts

```diff
@@ -89,6 +89,30 @@ function usesBasicForIam(obj: any): boolean {
   return obj.username === 'apikey' && !obj.password.startsWith('icp-');
 }
 
+// returns true if the string has a { or " as the first character
+// these are common user-issues that we should handle before they get a network error
+function badFirstChar(value: string): boolean {
+  return value.startsWith('{') || value.startsWith('"');
```

Comment:
Can you check for `}` and `"` as the last character as well here? and change the name to reflect that